terraform-plans/modules/GitHub/settings/main.tf

```diff
@@ -33,6 +33,25 @@ resource "github_repository" "repo" {
 
 }
 
+resource "github_actions_repository_permissions" "repo" {
+  repository      = var.repository
+  allowed_actions = "selected"
+
+  allowed_actions_config {
+    github_owned_allowed = true
+
+    verified_allowed = false
+
+    patterns_allowed = [
+      "beliaev-maksim/github-to-jira-automation/.github/workflows/issues_to_jira.yaml@master",
+      "canonical/*",
+      "snapcore/*",
```

Comment:
All repositories in https://github.com/snapcore have an equivalent in github.com/canonical/ - can we find out from the snap / store team which are the canonical sources?

Reply:
Sorry, I think I didn't understand your question...

So based on what you said, we can remove the `snapcore/*`?

Reply:
I think so, but some of our projects still use the `snapcore/*` versions. We could check and standardise. For now this is probably fine.